src/main/java/com/neuvector/Scanner.java

```diff
@@ -80,9 +67,12 @@ public static ScanRepoReportData scanRegistry(Registry registry, NVScanner nvSca
                     .withEnvironment("SCANNER_REGISTRY_USERNAME=" + registry.getLoginUser())
                     .withEnvironment("SCANNER_REGISTRY_PASSWORD=" + registry.getLoginPassword());
             }
-            String[] cmdArgs = builder.buildForImage(getNVImagePath(nvScanner.getNvScannerImage(), nvScanner.getNvRegistryURL()));
+            builder.withImage(getNVImagePath(nvScanner.getNvScannerImage(), nvScanner.getNvRegistryURL()));
+            builder.withCommand("/usr/local/bin/monitor && echo 'scan_result.json' && cat /var/neuvector/scan_result.json");
```

Comment:
`/usr/local/bin/monitor` is the default entrypoint. This is what does the container scan. Then we echo `scan_result.json` (so we can capture the output easily) and then print the contents of the `scan_result.json` in the shell. 

We will capture the printed value and pass it back to `insight-scanner`.